deploy/kubernetes/postee.yaml

```
@@ -188,6 +195,8 @@ spec:
               mountPath: /config
             - name: rego-template
               mountPath: /server/rego-template/custom
```

Use folder with suffix 's':
```suggestion
              mountPath: /server/rego-templates/custom
```